ambry-store/src/main/java/com/github/ambry/store/CompactionLog.java

```diff
@@ -479,5 +513,26 @@ safeToken if not null (see StoreFindToken#toBytes())
       bufWrap.put(safeTokenBytes);
       return buf;
     }
+
+    @Override
+    public String toString() {
+      StringBuffer buffer = new StringBuffer();
+      buffer.append("CompactionDetal:")
+          .append(compactionDetails.toString())
+          .append("\n")
+          .append("CopyStartTime:")
+          .append(copyStartTime)
+          .append("\n")
+          .append("CommitStartTime:")
+          .append(commitStartTime)
+          .append("\n")
+          .append("CleanupStartTime:")
+          .append(cleanupStartTime)
+          .append("\n")
+          .append("CycleEndTime:")
+          .append(cycleEndTime)
+          .append("\n");
```

Comment:
Probably we also need to check if token exists and append it to string if present.